pkg/skaffold/debug/transform_go.go

```diff
@@ -207,7 +217,12 @@ func (spec dlvSpec) asArguments() []string {
 	if spec.headless {
 		args = append(args, "--headless")
 	}
-	args = append(args, "--continue", "--accept-multiclient")
+
+	if !spec.wait {
```

Comment:
on L178, if the arg is "continue" you set spec.wait to false. Then here, if the value is false, you append the "--continue" arg. I'm trying to figure out why you need to do both. In other words, if spec.Wait is false, why does the --continue arg need to be added? Isn't that the default behavior?

Reply:
I am not getting entirely what you get. In the previous code you start from an image entrypoint/cmd and you instantiate accordingly a `dlvSpec` (we don't hold the args inside the `dlvSpec`). Where here you generate from a `dlvSpec` the appropriate cli command with args. This logic was already in place and not introduced by this PR 🙂